app/admin/cdr/cdr_exports.rb

```diff
@@ -225,6 +271,26 @@ def build_new_resource
                input_html: { class: 'chosen' },
                required: false
       ff.input :term_gw_external_id_eq, required: false
+      ff.input :customer_auth_external_type_eq, required: false
+      ff.input :customer_auth_external_type_not_eq, required: false
+      ff.input :customer_auth_external_id_in,
+               as: :select,
+               multiple: true,
+               collection: CustomersAuth.order(:name).pluck(:name, :external_id),
```

Comment:
could it be implemented using Ajax? possible big amount of customers auth records